ipatests/test_xmlrpc/test_schema_plugin.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Test the `ipaserver/plugins/schema.py` module.
+"""
+
+import pytest
+
+from ipalib import errors
+from ipatests.test_xmlrpc.tracker.base import Tracker
+from ipatests.test_xmlrpc.xmlrpc_test import XMLRPC_test
+
+
+@pytest.mark.tier1
+class TestParamFindAndShowCommand(XMLRPC_test):
+    """Test functionality of the ipa param-{find,show} command"""
+
+    tracker = Tracker()
+
+    def test_param_find(self):
+        """Test param-find command"""
+
+        # right command without criteria
+        result = self.tracker.run_command('param_find', u'user-add')
+        assert len(result['result']) != 0, result
+        expected = {'cli_name': 'login', 'name': 'uid', 'label': 'User login'}
+        assert set(expected.items()).issubset(set(result['result'][0].items()))
```

Comment:
@rcritten if you think this line is too complicated; I can change it with separate comparisons, like:
```python
assert result['result'][0]['cli_name'] == 'login'
assert result['result'][0]['name'] == 'uid'
```

Reply:
Ideally it would be broken out so in case of failure it would be clear which value failed the assertion.